common/lib/xmodule/xmodule/course_module.py

```diff
@@ -740,6 +740,16 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    enable_unproctored_exams = Boolean(
```

Comment:
Ultimately fix function name to match improved language.  It's too bad we don't have a good place for these settings.  I'll make sure that's on our list if it isn't already.